app/containers/ProfilePage/index.js

```diff
@@ -29,16 +35,19 @@ export class ProfilePage extends React.Component { // eslint-disable-line react/
 }
 
 ProfilePage.propTypes = {
-  dispatch: PropTypes.func.isRequired,
+  profile: PropTypes.object,
```

Comment:
We can but I'm not sure what the shape of that object will be. So this is just a placeholder.